### PR TITLE
Warn users about the dead events in case the events are _indeed_ dead.

### PR DESCRIPTION
**Pull request by armiol**

### CODE REVIEW

**Comment by armiol** [comment]
@alexander-yevsyukov PTAL.

File: server/src/main/java/org/spine3/server/event/EventBus.java [hunk 1]
```diff
@@ -222,23 +226,27 @@ private Event enrich(Event event) {
         return enriched;
     }
 
-    private void callDispatchers(Event event) {
+    /** Call dispatchers for the {@code event} passed and return the number of dispatchers called */
```
Comment (alexander-yevsyukov) [inline server/src/main/java/org/spine3/server/event/EventBus.java]: Pease document the value returned.

File: server/src/main/java/org/spine3/server/event/EventBus.java [hunk 2]
```diff
@@ -222,23 +226,27 @@ private Event enrich(Event event) {
...
         final Collection<EventDispatcher> dispatchers = dispatcherRegistry.getDispatchers(eventClass);
         for (EventDispatcher dispatcher : dispatchers) {
             dispatcher.dispatch(event);
+            dispatchersCalled++;
         }
+        return dispatchersCalled;
     }
 
-    private void invokeSubscribers(Message event, EventContext context) {
+    /** Invoke subscribers to the {@code event} passed and return the number of subscribers invoked */
```
Comment (alexander-yevsyukov) [inline server/src/main/java/org/spine3/server/event/EventBus.java]: Please document the returned value.

File: server/src/main/java/org/spine3/server/event/EventBus.java (hunk 1 above)
Comment (armiol) [inline server/src/main/java/org/spine3/server/event/EventBus.java]: Done.

File: server/src/main/java/org/spine3/server/event/EventBus.java (hunk 2 above)
Comment (armiol) [inline server/src/main/java/org/spine3/server/event/EventBus.java]: Done.

**Pull request merged**